### ISSUE TITLE
folder copy issue 

### ISSUE THREAD

**Issue by Demisexx** [kjeymax/GDUPLOAD_BOT2#7]
When clonening. folders itss getting Seprated..subfolders get outside from main folder

**Comment by askfriends** [comment]
> When clonening. folders itss getting Seprated..subfolders get outside from main folder

yes that is what i have already mentioned long ago, but sadly OP hasnt done any workaround for that.
